src/DependencyInjection/CsaGuzzleExtension.php

```diff
@@ -123,4 +119,16 @@ private function processClientsConfiguration(array $config, ContainerBuilder $co
             }
         }
     }
+
+    private function findSubscriberIds(array $explicitlyConfiguredIds, ContainerBuilder $container)
+    {
+        $allNames = array_values(array_map(
+            function ($tags) { return $tags[0]['alias']; },
+            $container->findTaggedServiceIds(SubscriberPass::SUBSCRIBER_TAG)
+        ));
+
+        return array_filter($allNames, function ($name) use ($explicitlyConfiguredIds) {
+            return !isset($explicitlyConfiguredIds[$name]) || $explicitlyConfiguredIds[$name];
+        });
+    }
```

Comment:
Hum, rather than injecting the subscribers in the extension, we should do it in the compiler pass.

I would be in favor of dropping support for disabling specific subscribers in the configuration for `v1.3.0`. It's too much of a hastle to support, and offers too little.